routes/api/v1/users.js

```diff
@@ -19,9 +19,9 @@ const { userValidation } = require("../../../middlewares/validations");
 
 // routes
 router.get("/", coreAuth, catchErrors(users));
-router.post("/", coreAuth, userValidation, catchErrors(addUser));
-router.put("/:id", leadAuth, catchErrors(updateUser));
-router.delete("/:id", coreAuth, catchErrors(deleteUser));
+router.post("/add", coreAuth, userValidation, catchErrors(addUser));
+router.put("/approve/:id", leadAuth, catchErrors(approveUser));
```

Comment:
"/add", "/approve/:id" and "/delete/:id" are not explicitely required in routes as they are POST, PUT and DELETE requests already.
Please use "/" and"/:id" only or use POST and GET instead of PUT and DELETE respectively.

Reply:
ok! sure